src/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -256,6 +256,17 @@ public void onEnable()
 		this.customLogger = new CustomLogger();
         
 		AddLogEntry("Finished loading configuration.");
+
+                // Check we can run with this Minecraft version
+                if (!Bukkit.getVersion().startsWith("1.12")) {
+                    AddLogEntry(
+                            "WARNING: this GriefPrevention version may not work"
+                            + " with Bukkit API version "
+                            + Bukkit.getVersion()
+                            + " - download an appropriate GP version"
+                    );
+                    return;
```

Comment:
I think I did  this as a "it isn't supported, don't load", but that might be a bit excessive - warning that it might not work but letting them try is probably more user-friendly - in a lot of cases it will still work fine with a slightly newer Minecraft version that doesn't contain major changes, so it would be a bit nasty to refuse to try.